docs/available_commands.md

```diff
@@ -20,6 +20,7 @@ Currently, the command adds or modifies the following pieces of information:
   * *externalReferences* of type *vcs*
 * Generates a *bom-ref* for components which don't have one, yet. The *bom-ref* will be a GUID.
 * If the path to a folder with license text files is provided, the text will be included in the SBOM, if the license has the corresponding `name`.
+* If a `license.name` is similar to an SPDX-ID, it will be replaced, e.g. `{"license": {"name": "The Apache License, Version 2.0"}}` leads to `{"license": {"id": "Apache-2.0"}}`.
```

Comment:
I would mention the list with licensenames that is included in the tool. The current wording might suggest a check based on similarity based on a textanalysis (what we dont support yet), not a simple comparision with a list.